subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/initialization/ConfigurationCacheStartParameter.kt

```diff
@@ -51,6 +51,12 @@ class ConfigurationCacheStartParameter(
 
     val encryptionAlgorithm: String = options.getInternalString("org.gradle.configuration-cache.internal.encryption-alg", SupportedEncryptionAlgorithm.AES_ECB_PADDING.transformation)
 
+    /**
+     * Should be provided if a link to the report is expected even if no errors were found.
+     * Useful in testing.
+     */
+    val alwaysLogReportLinkAsWarning: Boolean = options.getInternalFlag("org.gradle.configuration-cache.internal.report-link-as-warning", false)
```

Comment:
❌ I wonder if this shouldn't be a property instead (since it might never be called depending on what happens at runtime).